src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryCachingCleanup.java

```diff
@@ -8,29 +8,47 @@
 import java.io.IOException;
 import java.util.concurrent.TimeUnit;
 
+import jenkins.util.SystemProperties;
+
 @Extension public class LibraryCachingCleanup extends AsyncPeriodicWork {
-    private final Long recurrencePeriod;
-    private final Long unreadCacheClearTime;
+    public static /* non-final for script console */ int EXPIRE_AFTER_READ_DAYS =
+            SystemProperties.getInteger(LibraryCachingCleanup.class.getName() + ".CACHE_EXPIRY_DAYS", 7);
 
     public LibraryCachingCleanup() {
         super("LibraryCachingCleanup");
-        recurrencePeriod = Long.getLong("jenkins.workflow-libs.cacheCleanupPeriodDays", TimeUnit.DAYS.toMillis(7));
-        unreadCacheClearTime = Long.getLong("jenkins.workflow-libs.unreadCacheClearDays", TimeUnit.DAYS.toMillis(7));
```

Comment:
These did not follow standard Jenkins system property naming conventions, and I did not see any documentation related to them, so I created a new `EXPIRE_AFTER_READ_DAYS` property that follows the normal Jenkins style.